config/request-docs.php

```diff
@@ -58,13 +58,17 @@
         'description' => 'Laravel Request Docs',
         // default version that this library provides
         'version' => '3.0.0',
-        // changeable
+        //changeable
         'document_version' => '1.0.0',
         // license that you want to display
         'license' => 'Apache 2.0',
         'license_url' => 'https://www.apache.org/licenses/LICENSE-2.0.html',
         'server_url' => env('APP_URL', 'http://localhost'),
-
+        //openapi 3.0.x doesn't support request body for develop operation
+        //ref: https://github.com/OAI/OpenAPI-Specification/pull/2117
+        'include_body_on_delete' => false,
```

Comment:
```suggestion
        'delete_with_body' => false,
```

Reply:
Okay. I will commit this change

Reply:
I have push those corrections.[Commit Details](https://github.com/hafijul233/laravel-request-docs/commit/669a0333d8c8edde47260dc3e53c9fbe962ebc09)